src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -230,7 +231,7 @@ class PrisonerIepLevelReviewService(
           val levelCodeBeforeTransfer =
             iepHistory.sortedBy(IepDetail::iepTime).lastOrNull { it.agencyId != prisonerInfo.agencyId }?.iepCode
               ?: defaultLevelCode // if no previous prison
-          iepLevelService.findNearestHighestLevel(prisonerInfo.agencyId, iepLevelsForPrison, levelCodeBeforeTransfer)
+          iepLevelService.findNearestHighestLevel(prisonerInfo.agencyId, prisonIncentiveLevels, levelCodeBeforeTransfer)
```

Comment:
Similarly, the list of active prison levels could happen inside that service/method.

It would simplify the method invocation, but maybe it's annoying to change?